public/components/custom_panels/home.tsx

```diff
@@ -92,6 +93,7 @@ export const Home = ({
   const createCustomPanel = (newCustomPanelName: string) => {
     if (!isNameValid(newCustomPanelName)) {
       setToast('Invalid Operational Panel name', 'danger');
+      history.goBack();
```

Comment:
I missed this edge case - I will add it in a following PR, but I am wondering what does UX look like here - if somebody tries creating a panel from dashboards but then it fails because its an invalid name, do we want to land them back to the page before? Would toast even show up? Or what exactly is the process here

Reply:
If someone tries to create a panel but ends up with invalid name error, they should always be redirected to panels home page or else the create panel functionality breaks and the user won't be able to create a panel without manually refreshing or redirecting back to original page.  So I chnaged history.goback to window.location.assign so that if there's no history to go back to, it would land on panels home page and also not break the create panel functionality

Reply:
ideally i think we should validate the input at creating page, so whenever user click create, the name should be valid.